pkg/volume/cinder/cinder.go

```diff
@@ -397,8 +403,7 @@ func makeGlobalPDName(host volume.VolumeHost, devName string) string {
 }
 
 func (cd *cinderVolume) GetPath() string {
-	name := cinderVolumePluginName
-	return cd.plugin.host.GetPodVolumeDir(cd.podUID, kstrings.EscapeQualifiedNameForDisk(name), cd.volName)
+	return getPath(cd.podUID, cd.volName, cd.plugin.host)
```

Comment:
I am not comfortable with this change. Do they even point to the same directory?

Reply:
if you check that getPath function in rows 82-85 it will do the same thing. It is just common function for all of these getpath calls.

Reply:
cool